data/json/items/comestibles/egg.json

```diff
@@ -981,45 +1097,5 @@
     "copy-from": "egg_frog",
     "description": "A large clump of the eggs of a toad.  The eggs themselves are gelatinous and transparent.",
     "rot_spawn": "GROUP_EGG_TOAD"
-  },
-  {
-    "type": "COMESTIBLE",
-    "id": "egg_water_beetle",
```

Comment:
Don't reorder entries in files.

Reply:
I did this to put it all in one place and because, truthfully, I did not know that reordering entries messes with git blame. I'll revert this in a bit.

Reply:
Sorry, I know the rules are arcane, thanks for putting up with it.